src/ui_interface.h

```diff
@@ -130,4 +134,20 @@ bool InitError(const std::string& str);
 
 extern CClientUIInterface uiInterface;
 
+/** Show bilingual error message **/
+template <typename... Args>
+bool InitError(const bilingual_str& fmt, const Args&... args)
```

Comment:
I'd prefer if you removed the plain `InitError`, since it doesn't seem to be required.

Also, would be nice to add `InitError` to the linter `./test/lint/lint-format-strings.sh`

Reply:
> I'd prefer if you removed the plain InitError, since it doesn't seem to be required.

If I understand you correctly then what about https://github.com/bitcoin/bitcoin/pull/15340#discussion_r253929376?
Also see: fea324f90e58f74ad76045bd68db1f96f4b00200

Reply:
I mean that `InitError` could take as first argument a string or bilingual string, so that the formatting always happens inside of `InitError`. Then add `InitError` to the linter.